dev/io.openliberty.microprofile.openapi.ui.internal_fat/fat/src/io/openliberty/microprofile/openapi/ui/internal/fat/tests/UIOauthTest.java

```diff
@@ -118,6 +125,9 @@ public void teardownTest() throws Exception {
         //close the browser before stopping the server to reduce changes of open connections
         driver.quit();
         server.stopServer();
+
+        // Restore Config to original after test to prevent potential config bleeding between tests
+        server.updateServerConfiguration(baseConfig);
```

Comment:
This should probably go in a try-finally to ensure we still reset the config if there were errors in the server logs (which will cause `stopServer()` to throw an exception).